notebooks/00_quick_start/sar_movielens_with_azureml.ipynb

```diff
@@ -115,59 +117,33 @@
     "\n",
     "An [AzureML Workspace](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.workspace.workspace?view=azure-ml-py) is an Azure resource that organizes and coordinates the actions of many other Azure resources to assist in executing and sharing machine learning workflows. In particular, an Azure ML Workspace coordinates storage, databases, and compute resources providing added functionality for machine learning experimentation, deployment, inferencing, and the monitoring of deployed models.\n",
     "\n",
-    "**If you are trying to create or connect to a workspace for the first time in this environment, uncomment and run the below cell. ** It will load code necessary to create workspace. After code is loaded, run the cell again to create the workspace. It will create `aml_config\\config.json` file which contains workspace configuration for future notebook runs. See [this tutorial](https://docs.microsoft.com/en-us/azure/machine-learning/service/setup-create-workspace#portal) to locate information such as subscription id."
+    "The function below will get or create an AzureML Workspace and save the configuration to `aml_config/config.json`. It defaults to use existing configuration file, but falls back retrieving a workspace based on parameters provided as inputs or environment variables. Lastly, if the workspace does not exist, one will be created for you. See [this tutorial](https://docs.microsoft.com/en-us/azure/machine-learning/service/setup-create-workspace#portal) to locate information such as subscription id."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "You are currently using version 1.0.21 of the Azure ML SDK\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "print(\"You are currently using version\", azureml.core.VERSION, \"of the Azure ML SDK\")"
+    "ws = get_or_create_workspace()"
```

Comment:
We chose to do it with the magic load so that users do not have to leave this notebook to add the subscription info or resource group. This will then create the config.json. If we have a utility function, they will still need to go elsewhere to upload a config.json.

Reply:
this function works the same way as the load script, if the user already has a config it will use it, otherwise it will use environment variables (or inputs to the method) to look up the workspace, and lastly it will create the workspace with the settings if needed. in all cases the config.json is saved locally and the user isn't required to go to the portal to retrieve them.

Reply:
Oh I see, I didn't realize you could put parameters. Can we explicitly show the parameters and explain that if it is a user's first time they can fill them in? `get_or_create_workspace(
    config_path=None,
    subscription_id=None,
    resource_group=None,
    workspace_name=None,
    workspace_region=None,
)`

Reply:
sure makes sense, i'll add a comment that the user either needs to have config.json already, or set the env vars, or pass them in.